polling_stations/apps/file_uploads/templates/file_uploads/index.html

```diff
@@ -0,0 +1,178 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block extra_page_css %}
+<style>
+  button:disabled {
+    background-color: lightgrey;
+    border-color: darkgrey;
+  }
```

Comment:
It would be quite nice to have a disabled button style in the base theme. In general there are a few "trying to build a CRUD app with the v3 theme" issues I hit with this PR. I hacked this one, but it really tested the limits of my CSS skills ;)